projects/plugins/boost/app/assets/src/js/features/image-cdn/quality-settings/quality-settings.tsx

```diff
@@ -54,6 +54,7 @@ const QualitySettings = ( { isPremium }: QualitySettingsProps ) => {
 					closeEditText={ __( 'Hide', 'jetpack-boost' ) }
 					header={ <Header /> }
 					summary={ <Summary imageCdnQuality={ imageCdnQuality } /> }
+					tracksEventName="image_cdn_adjust_quality_clicked"
```

Comment:
I think this should be `image_cdn_panel_toggle` to match the pattern we've been using so far with the rest of the UI. Currently it's tied to the name and the fact that it's clickable.

For example:
- cornerstone_pages_panel_toggle
- performance_history_panel_toggle

```suggestion
					tracksEventName="image_cdn_panel_toggle"
```

Reply:
Yup, good suggestion, thanks.